Interni/NormeProgetto/components/norme-c3.tex

```diff
@@ -0,0 +1,475 @@
+\end{itemize}
+\subsubsection{Istanziazione di un processo}
+Con il perseguimento della qualità in mente, il gruppo \textit{\Gruppo{}} ha deciso di adottare le seguenti regole al momento della definizione di ogni processo, in modo da curare la qualità sin dall'istanziazione dei processi:
+\begin{itemize}
+	\item un singolo processo deve essere più atomico possibile;
+	\item l'obiettivo di un processo non può entrare in conflitto con quello di altri processi;
+	\item l'affidamento di risorse umane,temporali e materiali ai singoli processi deve tenere conto tutti gli altri processi e deve puntare ad essere più efficiente possibile;
+	\item ogni processo deve essere pianificato correttamente tramite una opportuna analisi dei rischi;
+	\item ogni processo al termine della pianificazione deve avere una durata ben definita.
+\end{itemize}
+\subsection{Processo di verifica}
+\subsubsection{Scopo}
+Il processo di verifica si occupa di determinare se i prodotti di una data attività sono corretti, coesi e completi e di individuare eventuali errori introdotti nella fase di sviluppo. Sono soggetti a verifica sia il software che la documentazione.
+\subsubsection{Descrizione}
+Il processo di verifica si compone di due attività che permettono di verificare che il prodotto sia conforme alle aspettative di analisi (\S{}\ref{Analisi}) e di test (\S{}\ref{Test}). Esso viene applicato su ogni processo in esecuzione:
+\begin{itemize}
+	\item al raggiungimento di un livello di maturità sufficiente;
+	\item successivamente a cambiamenti inerenti lo stato.
+\end{itemize}
+\subsubsection{Aspettative}
+Il processo di verifica si occupa di prendere ciò che è stato prodotto e di restituirlo in uno stato conforme alle aspettative. Per assicurare il corretto funzionamento di tale processo devono essere rispettate alcune regole:
+\begin{itemize}
+	\item seguire procedure definite;
+	\item seguire criteri chiari ed affidabili;
+	\item i prodotti devono passare attraverso fasi successive opportunamente verificate;
+	\item il prodotto deve trovarsi in uno stato stabile dopo la verifica;
+	\item alla fine del processo deve essere possibile procedere alla fase di validazione.
+\end{itemize}
+\subsubsection{Attività}
+\paragraph{Analisi}
+\label{Analisi}
+Il processo di analisi si suddivide in Analisi statica ed Analisi dinamica.
+\subparagraph{Analisi statica}
+L'analisi statica permette di effettuare controlli su documenti e codice, verificando così che la corretta applicazione della correttezza (intesa come assenza di errori e difetti). Vi sono due metodologie per applicarla:
+\begin{itemize}
+	\item \textbf{Walkthrough:} tecnica di verifica consistente nella lettura da parte del team dell'intero documento o codice in cerca di anomalie. \textit{Verrà applicata principalmente nella fase iniziale del progetto, quando il team ancora non conosce i possibili errori che si possono incontrare}. Questa tecnica risulta molto onerosa in termini di efficienza ed efficacia.
```

Comment:
\textit in posizione strana. 
Probabilmente superflua e in contraddizione con le norme.